package.json

```diff
@@ -76,7 +76,7 @@
     "webpack-merge": "^3.0.0"
   },
   "scripts": {
-    "prepublish": "npm run clean && npm run build && npm run test",
+    "prepare": "npm run clean && npm run build && npm run test",
```

Comment:
I've changed it on master with another PR: https://github.com/theia-ide/theia/blob/master/package.json#L84